keyboards/dztech/dz65rgb/keymaps/yuannan/README.md

```diff
@@ -0,0 +1,70 @@
+[DZ65RGB/v3 Yuannan's Layout Imgur Album](https://imgur.com/a/BI2RjgE)
+
+![Alpha](https://i.imgur.com/ikqvrtF.png)
+![Vim Editing Layer](https://i.imgur.com/hoBvSRR.png)
+![Mouse and Autoclicker](https://i.imgur.com/QWmWaWY.png)
+![Hyper functions and RGB Layer](https://i.imgur.com/Zm6Tf6p.png)
+
+# Yuannan's DZ65 Custom Profile. Autoclicker, Vim, and more!
+
+## Features
+
+1. Much better mouse support. Now able to:
+	- Move mouse around with 3 levels of acceleration
+	- Click and nav using mouse buttons 1-5
+	- Mouse scroll in all 4 directions, autoclick with 5 different levels of speed!
+	- Increase or Decrease the speeds of {500, 50, 20, 10, 1} clicks per second \*(Read Notes). Press Shift while using the Increase or Decrease button to do the opposite. It works for any other keyboard not just the DZ65!
+2. Ctrl and Caps swapped. The new indicator is dynamically colored and has been moved to the CTRL key to reflect the keymap. 
+3. Improved navigation via Vim like keys.
+4. Dynamic Macros now enabled!
+5. Custom Editing Macros such as:
+	- Deleting entire line
+	- Deleting from cursor to start of line
+	- Deleting form cursor to end of line
+6. VIA/VIAL support.
```

Comment:
Vial isn't supported in the repo, so we generally remove references to it to avoid potential confusion. 
```suggestion
6. VIA support.
```